components/notebook-controller/README.md

```diff
@@ -35,19 +35,32 @@ All other fields will be filled in with default value if not specified.
 
 ## Environment parameters
 
-ADD_FSGROUP:  If the value is true or unset, fsGroup: 100 will be included
+ADD_FSGROUP: If the value is true or unset, fsGroup: 100 will be included
 in the pod's security context. If this value is present and set to false, it will suppress the
-automatic addition of fsGroup: 100 to the security context of the pod.  
+automatic addition of fsGroup: 100 to the security context of the pod.
```

Comment:
Since you opened this PR, can you also documents the CLI options in an extra section?
`metrics-addr` and `enable-leader-election`

Reply:
Update to include that.